snorkel/labeling/model/label_model.py

```diff
@@ -340,7 +340,7 @@ def get_accuracies(self) -> np.ndarray:
         -------
         >>> L = np.array([[1, 1, 1], [1, 1, -1], [-1, 0, 0], [0, 0, 0]])
         >>> label_model = LabelModel(verbose=False)
-        >>> label_model.fit(L)
+        >>> label_model.fit(L, n_epochs=1000, seed=123)
```

Comment:
Is there any other source of randomness when we set this seed? If not, we can drop the number of epochs and make the Ls more realistic.